Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN apt-get update && apt-get install -y ca-certificates build-essential clang o
 
 ENV XDG_CACHE_HOME="/tmp"
 
-### taken from https://github.com/rust-lang/docker-rust/blob/master/1.63.0/buster/Dockerfile
+### taken from https://github.com/filecoin-project/lotus/blob/master/Dockerfile#L63
```

Comment:
```suggestion
```

just remove it

Reply:
Because this link is self-referential, `master`, and doesn't point to anything meaningful, I guess this is coming from AI tooling? Just deleting it is the right answer here. thanks for finding it.